civis/tests/test_io.py

```diff
@@ -320,7 +320,8 @@ def test_civis_file_to_table_table_doesnt_exist(
         self.mock_client.default_credential = 713
 
         self.mock_client.get_table_id.side_effect = ValueError('no table')
-        mock_columns = [{'name': 'foo', 'sql_type': 'INTEGER'}]
+        mock_columns = [{'name': 'foo', 'sql_type': 'INTEGER'},
+                        {'name': 'bar', 'sql_type': 'VARCHAR(42)'}]
```

Comment:
I added a second, different sql type to show that this test passes with current logic